### DISCUSSION TITLE
How to specify the UTF-8 encoding?

### DISCUSSION THREAD

**Discussion by JinXiu2023** [open-policy-agent/community#434] [OPA and Rego] [answered]
Hello, I have a question about encoding. I wonder whether the built-in function `concat()` produce a UTF-8 encoding string or not. And how to specify the UTF-8 encoding string if  the `concat()` doesn't give a UTF-8 encoding string?

Thanks for your answers.

**Comment by anderseknert** [comment, marked as answer]
OPA only supports UTF-8, so that’s the default. If anything doesn’t work with UTF-8, that’s likely a bug.

**Reply by JinXiu2023** [reply]
Ok. Thank you.
